### DISCUSSION TITLE
Crazyflie project

### DISCUSSION THREAD

**Discussion by BruceGlover** [bitcraze/discussions#957] [Q&A] [answered]
Hey all, I am new to drones. Wanted to build some projects to get the gist of the tech behind. Have some crazyflie drones and already got some tests done via the code samples in the repo. Could I get some project examples involving swarms and some new ideas as well.

**Comment by knmcguire** [comment, marked as answer]
Hi!

You can get started with the step-by-step guides to get some basics: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/. That ends up as well with a swarming example too.

After that you can try out some of the autonomy examples or swarming examples in the repository example folder: https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/

There is also the crazyswarm2 project that uses ROS2 https://imrclab.github.io/crazyswarm2/. 

Happy hacking!

**Reply by BruceGlover** [reply]
Thanks a lot. I will check them and ask some more questions too in the process.

**Reply by knmcguire** [reply]
sure thing! Here in discussions it is usually easier to ask one question per discussion, so you can go ahead and select this answer and start new discussion threads along the way of your process

**Reply by amitx64** [reply]
Hi, whenever I try to build the `example app` for the `platform-bolt`, it always gets built for the `cf2 platform`.

**Reply by knmcguire** [reply]
hi! please make a new discussion thread for your question
